gateway/conf/nginx.conf.liquid

```diff
@@ -5,12 +5,16 @@ env RESOLVER;
 env BACKEND_ENDPOINT_OVERRIDE;
 env OPENSSL_VERIFY;
 
+{% if jaeger_enabled %}
+load_module /opt/app-root/src/nginx-modules/ngx_http_opentracing_module.so;
+load_module /opt/app-root/src/nginx-modules/ngx_http_jaeger_module.so;
+{% endif %}
+
 daemon {{ daemon | default: 'off' }};
 master_process {{ master_process | default: 'on' }};
 worker_processes {{ worker_processes | default: 'auto' }};
 pcre_jit on;
 pid {{ pid | default: 'nginx.pid' }};
-timer_resolution {{ timer_resolution | default: '100ms' }};
```

Comment:
This should be disabled only when opentracing is enabled. Or the timer_resolution should be set to `0` (if that works).